File: python/class 5/question 6.py
mark_1=float(input('Enter your first mark: '))
if mark_1>100 or mark_1<0:
  print('Invalid test score.')
  exit()
mark_2=float(input('Enter your second mark: '))
if mark_2>100 or mark_2<0:
  print('Invalid test score.')
  exit()  
mark_3=float(input('Enter your third mark: '))
if mark_3>100 or mark_3<0:
  print('Invalid test score.')
  exit()
average=(mark_1+mark_2+mark_3)/3
if average>=50:
  print('Pass')
else:
  print('Fail')

File: java/class 5/Question6.java
import java.util.Scanner;

public class Question6{
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter your first mark: ");
    int mark1 = input.nextInt();
    System.out.print("Enter your second mark: ");
    int mark2 = input.nextInt();
    System.out.print("Enter your third mark: ");
    int mark3 = input.nextInt();
    int grade = (mark1+mark2+mark3)/3;

    if (0 > grade || grade > 100) {
      System.out.println("Invalid");
    } else {
        if (50 <= grade && grade <= 100){
          System.out.println("pass");
        } else{
          System.out.println("fail");
        } 
      }
    input.close();
  }
}
